packages/cli/build.gradle.kts

```diff
@@ -1424,12 +1422,12 @@ tasks {
   }
 
   dockerfileNative {
-    graalImage = "${project.properties["elide.publish.repo.docker.tools"]}/gvm21:latest"
+    graalImage = "${project.properties["elide.publish.repo.docker.tools"]}/gvm22:latest"
```

Comment:
note to self: `gvm22` image doesn't exist yet (code is there, hasn't been built and pushed)